src/main/java/me/zeroeightsix/kami/module/modules/chat/AutoExcuse.kt

```diff
@@ -41,10 +47,28 @@ object AutoExcuse : Module() {
         listener<PacketEvent.Receive> {
             if (mc.player == null || it.packet !is SPacketUpdateHealth) return@listener
             if (it.packet.health <= 0f && timer.tick(3L)) {
-                MessageSendHelper.sendServerMessage(getExcuse())
+                    MessageSendHelper.sendServerMessage(getExcuse())
             }
         }
     }
 
+    override fun onToggle() {
```

Comment:
shouldn't this only be onEnable()? I understand you want to reset the excuses back to `backup`, but you don't need to do it disabling, as enabling will do it when the user needs it.